tensorflow_io/hadoop/python/kernel_tests/hadoop_test.py

```diff
@@ -48,7 +49,7 @@ def test_sequence_file_dataset(self):
 
     dataset = hadoop_dataset_ops.SequenceFileDataset(filenames).repeat(
         num_repeats)
-    iterator = dataset.make_initializable_iterator()
+    iterator = dataset_ops.make_initializable_iterator(dataset)
```

Comment:
I guess it would be better to use `tf.compat.v1.data.make_one_shot_iterator(dataset)` as it's mentioned in https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/data/ops/dataset_ops.py#L1358. What do you think?

Reply:
Thanks @dmitrievanthony. Updated and using `tensorflow.compat.v1.data` now.